### PR TITLE
Update Narwhal readme and code owners

### PR DESCRIPTION
**Pull request by mwtian**
To make them work in the new repo.

### CODE REVIEW

File: narwhal/README.md [hunk 1]
```diff
@@ -1,21 +1,15 @@
-# Narwhal and Tusk
+# Narwhal and Conesensus
```
Comment (akichidis) [inline narwhal/README.md:1]: typo: `Conesensus --> Consensus`

**Review by akichidis — APPROVED** [review]
Thanks @mwtian for the update ! Are you planning to update the `CONTRIBUTING` on a later PR? Otherwise we could do it as part of this PR

File: narwhal/README.md (hunk 1 above)
Comment (mwtian) [reply narwhal/README.md]: Thanks, fixed.

**Comment by mwtian** [comment]
> Thanks @mwtian for the update ! Are you planning to update the `CONTRIBUTING` on a later PR? Otherwise we could do it as part of this PR

Updated in this PR.

**Pull request merged**